bucket/obsidian.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.9.4",
+    "description": "Obsidian is a powerful knowledge base that works on top of a local folder of plain text Markdown files.",
+    "homepage": "https://obsidian.md",
+    "license": "https://obsidian.md/eula",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/obsidianmd/obsidian-releases/releases/download/v0.9.4/Obsidian.0.9.4.exe#/dl.7z",
+            "hash": "db30e56d822792856605e398487fb807cdd54877829dbe7d9cd70f30a6907346",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Force -Recurse"
+                ]
+            }
+        }
+    },
+    "bin": "Obsidian.exe",
```

Comment:
What are available parameters?

Reply:
I have tried to call it with `-h`, `--help` and `/?`. It always just starts the UI, so I'm assuming it doesn't have any.

Reply:
Then remove the bin property

Reply:
I thought that is there for allowing to call it from the command line.
But probably few people will do that.